scripts/scripts.js

```diff
@@ -41,11 +41,129 @@ $(document).ready(function(){
                 var file = $('#fileid').get(0).files[0];
                 readBlob();
         });
+    InitScratchPad();
     ShowWorkArea(false);
 
     $('#undoDelete').hide();
 });
 
+function InitScratchPad() {

```

Comment:
Please consider storing the configuration of this to another file.

Reply:
ok